spec/rubocop/cop/layout/empty_lines_around_class_body_spec.rb

```diff
@@ -315,5 +313,39 @@ class Dad
     end
   end
 
+  context 'when EnforcedStyle is beginning_only' do
+    let(:cop_config) { { 'EnforcedStyle' => 'beginning_only' } }
+    let(:class_with_beginning_space_source) { <<-RUBY.strip_indent }
```

Comment:
It seems a little weird to me to treat a heredoc as a single line block body. I think using `do / end` would be more appropriate.

```ruby
let(:class_with_beginning_space_source) do
 <<-RUBY.strip_indent
    class SomeClass

      do_something
    end
  RUBY
end
```

Reply:
I see your point. I just followed what I thought was the convention.

Searching with `rg "<<-[A-Z_]+\s?[)}]"` reveals 123 uses of a heredoc being enclosed by `)` or ` }` and having the delimiting identifier on a subsequent line.

When I quickly scanned through the source, I couldn't find a heredoc within a `do / end`

Reply:
Thanks for usage info. Since there is already a precedence for this style, I am fine with it as is. 

Using heredocs inside of `()` is different than using them inside of `{}`. There is no multi-line form of `()` like there is with `{}` and `do / end`. I have become used to seeing heredocs inside of `()`. Seeing them in `{}` threw me for a second.